### PR TITLE
Russian translation for Symbol Explorer

### PR DESCRIPTION
**Pull request by klimgeran**

### CODE REVIEW

File: src/config/i18n/ru.json [hunk 1]
```diff
@@ -1,153 +1,200 @@
...
     "account": "Аккаунт",
     "block": "Блок",
     "mosaic": "Мозаика",
     "namespace": "Пространство имен",
     "namespaceName": "Имя",
-    "transaction": "Транзакция",
-    "transactionHash": "Хэш Транзакции",
-    "transactionId": "(ID) Идентификатор Транзакции",
+    "transaction": "транзакция",
+    "transactionHash": "Транзакционный Хэш",
```
Comment (OlegMakarenko) [inline src/config/i18n/ru.json:26]:
```suggestion
    "transactionHash": "Хэш транзакции",
```

File: src/config/i18n/ru.json [hunk 2]
```diff
@@ -1,153 +1,200 @@
...
+    "hashAlgorithm": "Алгоритм хеширования",
     "secret": "Секрет",
-    "linkedNamespace": "Связанное Пространство имен",
-    "minApproval": "Минимальное Подтверждение",
-    "minRemoval": "Минимальное Удаление",
-    "nothingToShow": "Пусто.",
+    "linkedNamespace": "Связанное пространство имен",
+    "minApproval": "Минимальное одобрение",
+    "minRemoval": "Минимальное снятие",
+    "nothingToShow": "Нечего показывать.",
```
Comment (OlegMakarenko) [inline src/config/i18n/ru.json:75]:
```suggestion
    "nothingToShow": "Нечего показать.",
```

**Comment by klimgeran** [comment]
Thanks for correcting the text, I appreciate it! Already fixed @OlegMakarenko 🙂👍🏻

**Review by OlegMakarenko — APPROVED** [review]
Thank you

**Review by klimgeran — COMMENTED** [review]
@OlegMakarenko Added changes to the Russian version

**Review by klimgeran — COMMENTED** [review]
Updated Symbol Explorer (ru)
@OlegMakarenko

**Review by klimgeran — COMMENTED** [review]
Translation updates

**Review by OlegMakarenko — APPROVED** [review]

**Pull request merged**